### DISCUSSION TITLE
为什么输入视频，但是没有任何输出结果？

### DISCUSSION THREAD

**Discussion by caojiaxin913** [mikel-brostrom/yolo_tracking#1681] [Q&A] [answered]
输入视频，但是没有任何输出结果。

**Comment by 14790897** [comment, marked as answer]
首先自己读一下 tracking/track.py的源代码

要想显示，给你举个例子：
python tracking\track.py --yolo-model yolov8n.pt --source ./test.mp4 --tracking-method deepocsort --conf 0.5 --iou 0.7 --device 0 --show --save  --device=cpu
